mypy/myunit/__init__.py

```diff
@@ -129,6 +129,8 @@ def set_up(self) -> None:
     def tear_down(self) -> None:
         if self.suite:
             self.suite.tear_down()
+        assert self.old_cwd is not None and self.tmpdir is not None, ("test was not"
+                                                                      " properly set up")
```

Comment:
In this case I would prefer using a `\` line continuation so the string doesn't have to be broken up (broken-up strings are harder to grep for).